neurom/fst/_neuritefunc.py

```diff
@@ -96,6 +96,32 @@ def _seclen(sec, **kwargs):
     return map_sections(_seclen, neurites, neurite_type=neurite_type)
 
 
+def section_term_lengths(neurites, neurite_type=NeuriteType.all):
```

Comment:
I think a lot of this overlaps what was added in #595; section_term_lengths, section_bif_lengths, section_bif_branch_orders, partition_asymmetries.  How do these differ?

Reply:
Yes, you are correct. This code was implemented a while back (so these functions were not there yet). I think the important additions that are missing and @wizmer needs are the two functions about the trunk measurements that are used as synthesis input.

Reply:
I think it would probably be best to reduce the pull request to the parts that are needed, so it's clear what is being added